Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM registry.redhat.io/ubi8/ubi AS builder
+
+ENV S2I_GIT_VERSION="" \
+    S2I_GIT_MAJOR="" \
+    S2I_GIT_MINOR=""
+
+ENV GOPATH="/go" \
+    GOARCH="amd64"
+
+RUN yum install -y \
+    make \
+    git \
+    go
```

Comment:
all three of these should be unnecessary once you base off of the image @sbose78 noted above